lib/rules/callback-return.js

```diff
@@ -39,7 +39,15 @@ module.exports = function(context) {
      * @returns {Boolean} Whether or not this function matches our callback name.
      */
     function isCallback(node) {
-        return node.callee.type === "Identifier" && callbacks.indexOf(node.callee.name) > -1;
+        if (node.callee.type === "Identifier" && callbacks.indexOf(node.callee.name) > -1) {
+            return true;
+        }
+
+        if (node.callee.type === "MemberExpression" && callbacks.indexOf(node.callee.object.name + "." + node.callee.property.name) > -1) {
```

Comment:
Yeah, I just noticed that something like `reply( error.data ).code( 400 );` (from hapi) doesn't work either.